Detectors/FIT/FV0/simulation/include/FV0Simulation/FV0DigParam.h

```diff
@@ -33,19 +33,20 @@ struct FV0DigParam : public o2::conf::ConfigurableParamHelper<FV0DigParam> {
   float shapeN = 2.36408e+00;          // Crystal ball N     parameter
   float shapeSigma = 3.55445;          // Crystal ball sigma parameter
   //float timeShiftCfd = 3.3;          // From the cosmic measurements of FV0 [keep it for reference]
-  float timeShiftCfd = 5.3;            // TODO: adjust after FV0 with FEE measurements are done
-  float singleMipThreshold = 3.0;      // in [MeV] of deposited energy
-  float singleHitTimeThreshold = 120.0; // in [ns] to skip very slow particles
-  float waveformNbins = 10000;         // number of bins for the analog pulse waveform
-  float waveformBinWidth = 0.01302;    // number of bins for the analog
-  float avgCfdTimeForMip = 8.63;       // in ns to shift the CFD time to zero TODO do ring wise
+  float timeShiftCfd = 5.3;                                                   // TODO: adjust after FV0 with FEE measurements are done
+  float singleMipThreshold = 3.0;                                             // in [MeV] of deposited energy
+  float singleHitTimeThreshold = 120.0;                                       // in [ns] to skip very slow particles
+  UInt_t waveformNbins = 10000;                                               // number of bins for the analog pulse waveform
+  float waveformBinWidth = 0.01302;                                           // bin width [ns] for analog pulse waveform
+  float avgCfdTimeForMip = 8.63;                                              // in ns to shift the CFD time to zero TODO do ring wise
   int chargeIntBinMin = (avgCfdTimeForMip - 6.0) / waveformBinWidth;          //Charge integration offset (cfd mean time - 6 ns)
   int chargeIntBinMax = (avgCfdTimeForMip + 14.0) / waveformBinWidth;         //Charge integration offset (cfd mean time + 14 ns)
```

Comment:
just a word of caution: these lines will affect only default values, since this is a configurable param, changing the avgCfdTimeForMip of waveformBinWidth will not modify the derived values. Perhaps they should be calculated elsewhere?